src/app/shared/components/addscenario/addscenario.component.html

```diff
@@ -1,21 +1,31 @@
 <ng-template #addScenario let-c="close" let-d="dismiss">
     <div class="modal-header">
         <div class="col-xs-12">
-            <h4 class="modal-title pull-left">New Scenario </h4>
+            <h4 class="modal-title pull-left" *ngIf="clone==true"> Clone Scenario </h4>
+            <h4 class="modal-title pull-left" *ngIf="clone==false"> New Scenario </h4>
             <!--close and save-->
             <button class="close pull-right" (click)="c('closed')">x</button><br><br>
             
         </div>
     </div>
     <div class="modal-body">
         <form [formGroup]="newScenForm">
+
+            <!-- Clone - choose region -->
+            <div class="form-group" *ngIf="clone==true">
+                <label class="req" for="regionid">Region:</label>
+                <select class="form-control" formControlName="regionid">
+                    <option *ngFor="let r of regions" [ngValue]="r">{{r.name}}</option>
+                </select>
+            </div> 
```

Comment:
I think is fine, but I'm wondering if it could have been done outside of the form (so placed after line 11 instead), and just hooked up to adjust the `selectedRegion` variable, so you don't have to add it to, and remove it from, the form.